payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/FormLabel.kt

```diff
@@ -0,0 +1,19 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import com.stripe.android.ui.core.PaymentsTheme
+
+@Composable
+internal fun FormLabel(
```

Comment:
Where else are we using this? I only see this used here: https://github.com/stripe/stripe-android/pull/4722/files#diff-e7e111171d564d1f2b37445fe138b29073730c366778fd37e6da7f85c5ec83d4R70